integration_tests/README.md

```diff
@@ -58,6 +58,11 @@ All of the tests will run in a single application.  They just enable and disable
 
 You do need to have access to a compatible GPU with the needed CUDA drivers. The exact details of how to set this up are beyond the scope of this document, but the Spark feature for scheduling GPUs does make this very simple if you have it configured.
 
+### Runtime Environment
+
+`--runtime_env` is used to specify the environment you are running the tests in. Valid values are `databricks` and `emr`. This is generally used
+when certain environments have different behavior.
```

Comment:
nit: Can we detect `emr` at runtime? I know for databricks it is hard to detect so we avoid doing it directly in the tests.

I am fine with doing it on the command line for consistency.

Reply:
yeah I did it for consistency and if you noticed here we actually have another check for 3.0.0. I was going to file a follow on try to commonize as I think we could use the shim layer for it all now.

Reply:
https://github.com/NVIDIA/spark-rapids/issues/830